packages/kmp-sdk/shared/src/iosMain/kotlin/xyz/self/sdk/handlers/DocumentsBridgeHandler.kt

```
@@ -5,29 +5,122 @@
+            )
+        }
+
+    private fun loadCatalog(): JsonElement {
+        val provider =
+            SdkProviderRegistry.documents
+                ?: throw BridgeHandlerException("NOT_CONFIGURED", "Documents provider not configured")
+
+        val catalogJson =
+            try {
+                provider.loadCatalog()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)